server/tests/src/test/java/org/infinispan/server/functional/XSiteRestMetricsResource.java

```diff
@@ -0,0 +1,96 @@
+package org.infinispan.server.functional;
+
+import static org.infinispan.server.functional.XSiteIT.LON;
+import static org.infinispan.server.functional.XSiteIT.NYC;
+import static org.infinispan.server.test.core.Common.sync;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Map;
+
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestMetricsClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.commons.configuration.StringConfiguration;
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.server.test.junit4.InfinispanXSiteServerRule;
+import org.infinispan.server.test.junit4.InfinispanXSiteServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * Test site status metrics
+ *
+ * @since 14.0
+ */
+public class XSiteRestMetricsResource {
```

Comment:
```suggestion
public class XSiteRestMetricsResourceTest {
```

Reply:
Then I would suggest IT instead. The problem is with it just being Resource like this it screams it is an impl class for another test or something but in actuality runs tests.

Reply:
I can't add `IT` otherwise it runs in parallel with `XSiteIT` and breaks the cross-site connection 
```
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.144 s <<< FAILURE! - in org.infinispan.server.functional.XSiteRestMetricsResourceIT
[ERROR] org.infinispan.server.functional.XSiteRestMetricsResourceIT.testSiteStatus  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:12222
```

https://github.com/infinispan/infinispan/blob/main/server/tests/src/test/java/org/infinispan/server/functional/XSiteIT.java#L19-L23

Reply:
Alright, maybe `Operations` instead of `Resource`? That seems to be consistent with other tests.